src/dbnode/generated/thrift/generate.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# paranoia, ftw
+set -e
+
+# ensure docker is running
+docker run --rm hello-world >/dev/null
+
+# generate files using dockerized thrift-gen
+THRIFT_IMAGE_VERSION=${THRIFT_IMAGE_VERSION:-"quay.io/prateekrungta/thrift-gen:0.1.0"}
```

Comment:
Is there a better place to put/source this image?

Reply:
The source for this image is in the diff (`scripts/thriftgen.Dockerfile`) but I don't want to re-build it each time during execution. 

I'll open a followup to push the image to `quay.io/m3db/thriftgen` instead.

Reply:
#762